src/api/spec/controllers/webui/attributes_spec.rb

```diff
@@ -16,4 +16,20 @@
       expect(flash[:error]).to eq("Project not found: Does:Not:Exist")
     end
   end
+
+  describe 'POST #create' do
+    let(:attribute_type) { create(:attrib_type_with_namespace, value_count: 1) }
+
+    before do
+      login user
+      post :create, params: { attrib: { project_id: user.home_project.id, attrib_type_id: attribute_type.id } }
+    end
+
+    it {
+      expect(response).to redirect_to(edit_attribs_path(project: user.home_project_name,
+                                                        package: '',
```

Comment:
I meant to use `nil`, but doesn't make a difference and I am not sure if It would work. It is perfectly fine as it is. :wink: